server/raft.go

```diff
@@ -197,6 +197,9 @@ type raft struct {
 	leadc    chan bool
 	quit     chan struct{}
 
+	// Account name of the asset this raft group is for
+	accName string
```

Comment:
Maybe just make part of RaftConfig struct?

Reply:
No, because the place I need it (https://github.com/nats-io/nats-server/pull/2931/files#diff-0a3d4e9b87bb3880bdb735fa489ef618b571de3370dae31d8f509b26e2fb852eR2128), I have only `*raft` and I don't see we keep reference of the config.